test/Dialect/Tpp/tpp-ops.mlir

```diff
@@ -68,3 +68,17 @@ func.func @testBrgemmWithBf16(%arg0: memref<64x4x4xbf16>, %arg1: memref<64x2x4x2
   tpp.vnni_brgemm ins(%arg0: memref<64x4x4xbf16>, %arg1: memref<64x2x4x2xbf16>) out(%arg2: memref<4x4xbf16>)
   return %arg2: memref<4x4xbf16>
 }
+
+// CHECK-LABEL: func.func @addBcastColOperandOne
+func.func @addBcastColOperandOne(%arg0: memref<5xf32>, %arg1: memref<6x5xf32>) {
+  // CHECK: tpp.add
+  tpp.add ins(%arg0: memref<5xf32>, %arg1: memref<6x5xf32>) out(%arg1: memref<6x5xf32>)
+  return
+}
+
+// CHECK-LABEL: func.func @addBcastColOperandTwo
+func.func @addBcastColOperandTwo(%arg0: memref<6x5xf32>, %arg1: memref<5xf32>) {
+  // CHECK: tpp.add
+  tpp.add ins(%arg0: memref<6x5xf32>, %arg1: memref<5xf32>) out(%arg0: memref<6x5xf32>)
```

Comment:
Also test `<6x1xf32> -> <6x5xf32>` (ie. column broadcast). We use that in MHA.

Reply:
will add as a part of the unit test cases I am planning to add.